### PR TITLE
Lookat

### PR DESCRIPTION
**Pull request by cprogrammer1994**
- [x] Fix the function name so it's consistent with the rest of the library
- [x] Add tests
- [x] Fix dtype

### CODE REVIEW

File: pyrr/matrix44.py [hunk 1]
```diff
@@ -360,6 +362,30 @@ def create_orthogonal_projection_matrix(
...
+    forward = vector.normalise(eye - target)
+    side = vector.normalise(np.cross(up, forward))
+    up = vector.normalise(np.cross(forward, side))
+
+    return np.array((
+            (side[0], up[0], forward[0], 0.),
+            (side[1], up[1], forward[1], 0.),
+            (side[2], up[2], forward[2], 0.),
+            (-np.dot(side, eye), -np.dot(up, eye), -np.dot(forward, eye) , 1.0)
+        ), dtype=dtype)
```
Comment (cprogrammer1994) [inline pyrr/matrix44.py]: Lines 381 -> 386 looks strange to me.

My [code](https://github.com/cprogrammer1994/Headless-rendering-with-python/blob/master/matrices.py#L16) is a bit different.

I will create a few tests then we will see.

File: pyrr/matrix44.py [hunk 2]
```diff
@@ -360,6 +362,30 @@ def create_orthogonal_projection_matrix(
...
+
+    :param numpy.array eye: Position of the camera in world coordinates.
+    :param numpy.array target: The position in world coordinates that the
+        camera is looking at.
+    :param numpy.array up: The up vector of the camera.
+    :rtype: numpy.array
+    :return: A look at matrix that can be used as a viewMatrix
+    """
+
+    forward = vector.normalise(eye - target)
```
Comment (cprogrammer1994) [inline pyrr/matrix44.py]: this is "backward" rather then "forward"

**Comment by adamlwgriffiths** [comment]
I like the changes, thank you =)

Can you add one more test.
Create a look at matrix, let's say from [10, 0, 0] to [-10, 0, 0] with [0, 1, 0] as the up vector, and -z as forward [0, 0, -1].
then rotate a set of unit length x, y, z vectors ([1,0,0], [0,1,0],[0,0,1]) by the resultant matrix and check that they are as expected.
Ie, x should be facing [0, 0, -1], y should remain constant at [0, 1, 0], and z should be facing along +X [1, 0, 0].
I think that's right.

**Comment by adamlwgriffiths** [comment]
Nice!
May as well add yourself to the authors list in README.md and then we'll merge this in =)

**Comment by cprogrammer1994** [comment]
I will add myself

**Comment by adamlwgriffiths** [comment]
I've added you as a collaborator so you can go ahead and add yourself and merge in your own time.
I appreciate you taking the time to ensure your code is in keeping with the library.
And again, thanks for taking the time to work on this =)

**Comment by cprogrammer1994** [comment]
oops we will have a conflict then

**Comment by cprogrammer1994** [comment]
tests are passing on windows with python 3.5 and 3.6

**Comment by adamlwgriffiths** [comment]
Ah, I meant I gave your account collaborator permissions.
I haven't made any commits to the README.

Feel free to merge =)

**Pull request merged**